client/src/components/AuthNavigation.js

```diff
@@ -88,6 +88,22 @@ const AuthNavigation = () => {
         return history.push("/profile");
     }
 
+    const logout = async() => {
+        await fetch('/api/logout', {
+          method: 'GET',
+          headers: {
+            'Content-Type': 'application/json',
+          }})
+          .then((responce) => {
+            console.log(responce);
+          })
+          .catch((err) => {
+            console.log(err);
+          });
+
+        return history.push('/');
```

Comment:
you probably want to move it into the .then block to ensure the cookie is deleted

Reply:
Moved the return statement to the .then block.


Also removed the async declaration since a promise is being handled using .then bock